### DISCUSSION TITLE
OSX Issues

### DISCUSSION THREAD

**Discussion by CarliJoy** [CarliJoy/django-pint#27] [General]
@cornelv could you specify your issues with the local environment here?

The app is intended to be OS independent.
Was it a problem with getting postgresql (or the python connection) to run or getting the app itself to run with you local python?

I like having a also a docker container to develop but still would like to make sure everything is fine with the app on OS X.
Thing is I can't test it, as I don't have OSX.

**Comment by cornelv** [comment]
On my computer the issue is the python version, I installed 3.7 but it's not recognised, and when I do `pip install pipx` I get the error that Python 3.6 or later is required.  I am not sure how to get around it, that's why I prefer to have docker containers for all projects and avoid any dependency issues. 
should I investigate the issue deeper ?
